ledger-core/testutils/investigation/testrail_test.go

```diff
@@ -0,0 +1,234 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+package investigation
+
+// import (
```

Comment:
Will be uncommented later?

Reply:
No, it wont. It's was for testing purposes. Go doesn't support expected fail tests and other :(
I can remove it, if you suggest

Reply:
Let's remove it